homeassistant/components/vacuum/__init__.py

```diff
@@ -398,16 +398,21 @@ def add_to_platform_start(
                 "platform": self.platform.platform_name,
```

Comment:
Should we call `async_get_issue_tracker` to include a URL in the repair issue?

Reply:
Yes, that's a good idea! I'll add that.